### DISCUSSION TITLE
Plotting Intercative Map

### DISCUSSION THREAD

**Discussion by yamlakyam** [cu-esiil-edu/2024-data-short-course#52] [Q&A]
Searching for my school using the osm features, I got 2 records both of which are of POINT type. I want to ask how to extract the first from the results and if possible to get a polygon feature

**Comment by eculler** [comment]
@yamlakyam you can extract the first record using the following code, `.iloc`, which **locates** rows by the **index** value (that's the far left column ):

your_dataframe.iloc[0]

Remember that Python starts indexing at 0 and not 1!

To get a POLYGON feature, you would need to search for something that has a boundary. You can try tweaking your search to see if anything comes up as a POLYGON. What amenity type are you using? You can also try to search for a particular building.

You can try tweaking your search at [Open Street Maps online](https://www.openstreetmap.org/) for quicker feedback.
